fluent/syntax/serializer.py

```diff
@@ -235,7 +235,7 @@ def serialize_attribute_expression(expr):
 
 def serialize_variant_expression(expr):
     return "{}[{}]".format(
-        serialize_identifier(expr.id),
```

Comment:
Can you change this to `serialize_expression(expr.of)`?

Reply:
Follow-up for this landed.